### PR TITLE
Add option to disable codelens

### LINKED ISSUE #175: Disable Debug button
Is there any configuration to disable the debug button. It is making VS code behave strangely. First it adds a new line making it awkward to visualise my code. I have also experienced scrolling while I'm typing with the new debug functionality.

### PR DESCRIPTION
**Pull request by goncharov**
Resolves #175

### CODE REVIEW

File: src/extension.ts [hunk 1]
```diff
@@ -47,7 +47,13 @@ export function activate(context: vscode.ExtensionContext) {
     ...registerCoverageCodeLens(extensionInstance),
     registerToggleCoverageOverlay(pluginSettings.showCoverageOnLoad),
     vscode.commands.registerCommand(`${extensionName}.run-test`, extensionInstance.runTest),
-    vscode.languages.registerCodeLensProvider(languages, extensionInstance.codeLensProvider)
+    vscode.languages.registerCodeLensProvider(languages, extensionInstance.codeLensProvider),
+    vscode.workspace.onDidChangeConfiguration(e => {
```
Comment (orta) [inline src/extension.ts]: this is cool 👍

**Comment by goncharov** [comment]
Hello. Line endings was fixed

**Comment by orta** [comment]
Rock 👍

**Comment by orta** [comment]
Shipped with #188 as 2.5.2

**Comment by seanpoulter** [comment]
Would either of you know how to fix the [blame history](https://github.com/jest-community/vscode-jest/blame/master/src/JestExt.ts)? It's showing all the changes from the "fix the line endings" commit. A rebase to squash could work pre-merge -- do we have any options now?

**Comment by seanpoulter** [comment]
I'm looking at the blame history again and cursing. @goncharov, could you try rebasing your commits and we'll see if that'll update here and hopefully fix the history?

**Comment by orta** [comment]
This PR got merged @seanpoulter - not sure what your feedback is aimed at?

**Comment by seanpoulter** [comment]
Yea, sorry @orta. I realize now we'd have to revert, rebase to squash, and merge again. :confused: 

The feedback is aimed at cleaning up the [edit history](https://github.com/jest-community/vscode-jest/blame/master/src/JestExt.ts) for `JestExt.ts`. If you're using `git blame` to find the commits and PR for a block of code you end up with the two commits that changed the line endings.

As a lazy developer, I don't like having to find the button to look at the previous commit

image

when the history shows:

image

**Comment by orta** [comment]
Sure, you're welcome to do that - I don't really mind that kind of thing

**Pull request merged**